modules/@apostrophecms/piece-type/index.js

```diff
@@ -103,12 +103,24 @@ module.exports = {
         inputType: 'radio',
         unlessFilter: {
           archived: true
+        },
+        route: '/archive',
```

Comment:
These are batch operations, so I'm assuming we can base them on the piece type's action route. So we only need to the API route path as defined in the `apiRoute` customization function.

Reply:
I recognize this all crosses into batch operation work @ValJed  will be doing, but I needed something to test with.